schutzbot/deploy.sh

```diff
@@ -50,12 +51,21 @@ EOF
 source /etc/os-release
 ARCH=$(uname -m)
 
-if [[ -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
+if [[ $ID == "rhel" && $VERSION_ID == "8.3" && -n "${RHN_REGISTRATION_SCRIPT:-}" ]] && ! sudo subscription-manager status; then
     greenprint "Registering RHEL"
     sudo chmod +x "$RHN_REGISTRATION_SCRIPT"
     sudo "$RHN_REGISTRATION_SCRIPT"
 fi
```

Comment:
I believe you can just remove this instead of adding this condition. It's going to be ran only on 8.4 runners here and should already be subscribed.